docker/README.md

```diff
@@ -0,0 +1,56 @@
+
+# Docker-compose application for PHPRecipebook
+
+This docker-compose application starts two containers: a PostgreSQL server and
+an Apache web server with mod-php, where PHPRecipebook runs.
+
+## Minimal setup
+
+On the first run, enable PHPRecipebook's setup wizard.  Uncomment the
+environment variable setting `PHPRECIPEBOOK_SETUP_MODE=TRUE`.  After you've
+created your admin user turn it back off and restart the application to enable
+regular operation.
+
+## Usage
+
+### Starting it
+
+    docker-compose -f docker-compose.yml -d up
```

Comment:
docker-compose -f docker-compose.yml up -d

Put -d at the end

Reply:
Thanks @kkrieger85.  I just committed your change